src/software/sensor_fusion/sensor_fusion.h

```diff
@@ -77,11 +77,14 @@ class SensorFusion
     Team createEnemyTeam(const std::vector<RobotDetection> &robot_detections);
 
     /**
-     * Inverts all positions and orientations across the x and y axis of the field
+     *Inverts all positions and orientations across the x and y axis
      *
-     * @param frame The frame to invert. It will be mutated in-place
+     * @param Detection to invert
```

Comment:
I assume this is CI formatting causing inconsistent space b/w the '*' and the comments.